opentelemetry-sdk/benches/log.rs

```diff
@@ -27,6 +28,12 @@ fn log_benchmark_group<F: Fn(&dyn Logger)>(c: &mut Criterion, name: &str, f: F)
 
     group.bench_function("no-context", |b| {
         let provider = LoggerProvider::builder()
+            .with_config(Config::default().with_resource(Resource::new(vec![
```

Comment:
@lalitb I think we can the benchmarks as is because there is default Resource added anyway with 4 attributes. And show the gains for the default case. Also include a changelog entry, so folks upgrading knows that the upcoming version has noticeable perf gains.

Reply:
Done.